src/NServiceBus.Core/Settings/SettingsHolder.cs

```diff
@@ -231,6 +235,11 @@ public void Set<T>(Action value)
         /// </summary>
         /// <typeparam name="T">The type to use as a key for storing the setting.</typeparam>
         /// <param name="value">The setting value.</param>
+        [ObsoleteEx(
+            ReplacementTypeOrMember = "SetDefault<T>(T value)",
+            TreatAsErrorFromVersion = "9.0",
```

Comment:
Didn't notice this before and it's been a long time since I've looked at this stuff, but don't we want to Obsolete with Warning in 7.1, Error in 8.0 and Remove in 10? Or am I missing something?

Reply:
man...I even screwed up my numbers. Pay attention to @bording, not me @SeanFeldman

Reply:
It's ok, you were closer than me anyways